src/withLDConsumer.test.tsx

```diff
@@ -34,7 +34,7 @@ describe('withLDConsumer', () => {
 
   test('ldClient is passed down through context api', () => {
     const Home = (props: HocProps) => (
-      <div>{props.ldClient && props.ldClient.track ? 'ldClient detected' : 'Negative, no ldClient'}</div>
+      <div>{props.ldClient && props.ldClient.track() ? 'ldClient detected' : 'Negative, no ldClient'}</div>
```

Comment:
@yusinto What do you think of this fix? I do not know the context of this file

Reply:
The original code checks if `track` exists on the ldClient object, not invoking track so this change is incorrect. Were you able to reproduce the typescript error?